app/views/RoomView/index.tsx

```diff
@@ -785,13 +785,22 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 		});
 	};
 
+	handleCloseEmoji = (action?: Function, params?: any) => {
+		if (this.messagebox?.current) {
+			return this.messagebox?.current.closeEmojiAndAction(action, params);
+		}
+		if (action) {
+			return action(params);
+		}
+	};
+
```

Comment:
I needed to do this differentiation because there is the possibility that the message box isn't rendered, for example, when the room is `read only`.

So, in that case, besides the possibility to reactions on long press, the user must be able to click on the `emoji +` to add new emoji or long press on the reactions.